crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/spoof.rs

```diff
@@ -38,7 +19,7 @@ pub struct TxInfoMock {
     pub transaction_hash: Option<Felt252>,
     pub chain_id: Option<Felt252>,
     pub nonce: Option<Felt252>,
-    pub resource_bounds: Option<ResourceBounds>,
+    pub resource_bounds: Option<Vec<ResourceBounds>>,
```

Comment:
how come this is now vec and we don't need to change anything in cairo? 🤔

Reply:
Previously it was `ResourceBounds(pub Vec<Felt252>)` now it's `<Vec<ResourceBounds>>` where 
```rs
struct ResourceBounds {
    pub resource: Felt252,
    pub max_amount: u64,
    pub max_price_per_unit: u128,
}
```

Previously was this way to keep deserialising logic. Current is semantically correct

Reply:
Nothing changed in cairo, we just changed the interface in Rust (btw we aligned it with the cairo one). It still serializes to felts the same way